WindowsServerDocs/networking/windows-time-service/insider-preview.md

```diff
@@ -55,10 +54,9 @@ To address this problem, software timestamping allows us to timestamp packets be
 
 For more information, see:
 
--  Our [announcement blog](https://blogs.technet.microsoft.com/networking/2018/07/18/top10-ws2019-hatime/)
-
--  Validation Guide for the [IT Pro](https://github.com/Microsoft/SDN/blob/master/FeatureGuide/Validation%20Guide%20-%20RS5%20-%20Software%20Timestamping.docx)
+- Our [announcement blog](https://techcommunity.microsoft.com/t5/networking-blog/top-10-networking-features-in-windows-server-2019-10-accurate/ba-p/339739/)
 
+- Validation Guide for the [IT Pro](https://github.com/Microsoft/SDN/blob/master/FeatureGuide/Validation%20Guide%20-%20RS5%20-%20Software%20Timestamping.docx)
```

Comment:
@illfated While this link is technically correct, it goes one level too far to actually access the doc file. If you could change this to: https://github.com/microsoft/W32Time/tree/master/Leap%20Seconds, I'd appreciate it

Reply:
Sure thing, I was not aware that this link was an issue, but I have no problem changing it as you suggest. Thank you.

Reply:
By the way, if I had remembered to post the link to the **Hide whitespace changes** feature, you might have spotted that the github.com link was not among my initial changes.